src/Microsoft.AspNetCore.Server.HttpSys/NativeInterop/HttpApi.cs

```diff
@@ -70,724 +70,6 @@ internal static unsafe class HttpApi
         [DllImport(HTTPAPI, ExactSpelling = true, CallingConvention = CallingConvention.StdCall, SetLastError = true)]
         internal static extern unsafe uint HttpCloseRequestQueue(IntPtr pReqQueueHandle);
```

Comment:
These p-invokes should all be shared, no?

Reply:
Per offline feedback, these do not need to be shared.